atlasdb-commons/src/main/java/com/palantir/common/collect/IteratorUtils.java

```diff
@@ -69,12 +69,7 @@
      * </code>
      */
     public static <T> Iterable<T> forEach(final Iterator<T> it) {
```

Comment:
Sure. I'll also add a release note then, because it's used in one place in $internalCodebase and it'll break the bump until we inline this code there.